### PR TITLE
Extend Covid tracing demo

### PR DESCRIPTION
**Pull request by justinj**
This commit extends the Covid tracing demo to run in real-time. Includes a
bunch of randomly generated data to give it some texture.

### CODE REVIEW

File: covid_tracing/src/main.rs [hunk 1]
```diff
@@ -116,37 +127,49 @@ fn main() {
     df.add_edge(notifs_out, notifs_in);
 
     let reactor = df.reactor();
-    let (is_done, set_done) = std::sync::mpsc::sync_channel(0);
+
+    let all_people = people::get_people();
+    for person in &all_people {
+        people_send.send(person.clone()).unwrap();
+        reactor.trigger(people_op_id).unwrap();
+    }
```
Comment (MingweiSamuel) [inline covid_tracing/src/main.rs:135]: Can put this in another `std::thread::spawn` to emphasize the asynchrony?

**Review by MingweiSamuel — APPROVED** [review]

**Comment by MingweiSamuel** [comment]
And also run clippy ;P

File: covid_tracing/src/main.rs (hunk 1 above)
Comment (justinj) [reply covid_tracing/src/main.rs]: Done.

**Comment by justinj** [comment]
Done, thanks for the review!

**Pull request merged**